packages/react/src/themes/base/components/Provider/providerBoxVariables.ts

```diff
@@ -1,9 +1,31 @@
+import { pxToRem } from '../../../../lib'
+
 export interface ProviderBoxVariables {
   background: string
   color: string
+
+  scrollbarHeight: string
+  scrollbarWidth: string
+
+  scrollbarThumbBackgroundColor: string
+  scrollbarThumbBorderRadius: string
+  scrollbarThumbBorderSize: string
+
+  scrollbarThumbHoverBackgroundColor: string
+  scrollbarThumbHoverBorderSize: string
 }
 
 export default (siteVariables): ProviderBoxVariables => ({
   background: siteVariables.bodyBackground,
   color: siteVariables.bodyColor,
+
+  scrollbarHeight: pxToRem(16),
+  scrollbarWidth: pxToRem(16),
+
+  scrollbarThumbBackgroundColor: siteVariables.gray06,
```

Comment:
Oops, base theme doesn't have `gray06` and `gray04` :S

Reply:
Definitely, I initially overlooked the path of styles, now they are in correct place.

P.S. Motivation to add them only to Teams theme:
- one of the frequent questions in SUI: "How I can disable custom scroll bars?"
- this feature is not cross-browser

Reply:
I would really love to see this as part of base theme, but I see your points. Let's think about this in the future. The base theme's provider should have custom scrollbar by default, at least in my opinion. Agreed that we should have way of enabling or disabling this behavior as well. The main reason for this is that, I don't see much different ways of defining the scrollbar, more over we can customize it trough it's variables.